### PR TITLE
Add Integration Tests for Calculator Controller

### PR DESCRIPTION
**Pull request by hrogat**
Automated changes after successful tests.

## Agent Update

- **[Coder]** Created comprehensive integration tests for CalculatorController with MockMvc. Tests cover valid inputs, missing parameters, invalid parameters, and edge cases including negative numbers, large numbers, and integer boundaries.
- **[Tester]** All tests passed including the new CalculatorControllerIntegrationTest. The integration tests successfully verify the /sum endpoint with valid inputs, missing parameters, invalid parameters, and edge cases. Total tests run: 6, Failures: 0, Errors: 0, Skipped: 0.

### CODE REVIEW

File: src/test/java/com/tomwey2/calculator/CalculatorControllerIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+                .andExpect(status().isOk())
+                .andExpect(content().string("0"));
+
+        // Test with large numbers
+        mockMvc.perform(get("/sum")
+                        .param("a", "1000000")
+                        .param("b", "2000000"))
+                .andExpect(status().isOk())
+                .andExpect(content().string("3000000"));
+    }
```
Comment (hrogat2) [inline src/test/java/com/tomwey2/calculator/CalculatorControllerIntegrationTest.java]: please create one test for each call.

**Comment by hrogat** [comment]
**Automated Update:**

Automated changes after successful tests.

## Agent Update

- **[Coder]** Refactored CalculatorControllerIntegrationTest to have one test method per specific scenario as requested by the reviewer. The test coverage remains comprehensive with individual test methods for positive numbers, negative/positive combinations, zeros, large numbers, missing parameters, invalid parameters, and edge cases with max/min integer values.
- **[Tester]** All tests passed successfully. The refactored CalculatorControllerIntegrationTest now has 12 individual test methods covering all scenarios: positive numbers, negative/positive combinations, zeros, large numbers, missing parameters, invalid parameters, and edge cases with max/min integer values. Total tests run: 14, Failures: 0, Errors: 0, Skipped: 0.
